nix/nixos/cardano-node-service.nix

```diff
@@ -27,7 +27,7 @@ in {
       uid = 10016;
       group = "cardano-node";
     };
-    systemd.services.cardano-node = {
+    systemd.services."cardano-node@" = {
```

Comment:
we should optionally do this if it's a cluster vs just a single node

Reply:
Agreed.